lib/restart_game.dart

```diff
@@ -8,24 +8,24 @@ class RestartGame extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Directionality(
-        textDirection: TextDirection.rtl,
-        child: Center(
-            child: Column(
-                mainAxisSize: MainAxisSize.min,
-                children: [
-                  Text(
-                      'Perdeu o jogo',
-                      textDirection: TextDirection.rtl,
-                  ),
-                  RaisedButton(
-                      onPressed: this.onPressRestart,
-                      child: Text(
-                          'Reiniciar',
-                      )
-                  ),
-                ],
+      textDirection: TextDirection.rtl,
```

Comment:
For the game UI, Flame has a nice api called widgets overlay, which can make this logic of your way easier, I can explain it later if you want

Reply:
Yes! I want!